src/structure/grid.ts

```diff
@@ -513,7 +518,7 @@ export class ViewersGrid {
      */
     private _setupCell(cellGUID: GUID, colNum: number, rowNum: number): string {
         const cellId = `gi-${cellGUID}`;
-        let cell = document.getElementById(cellId);
+        let cell = this._root.querySelector(`#${cellId}`) as HTMLElement;
```

Comment:
```suggestion
        let cell = this._root.querySelector(`#${cellId}`) as HTMLElement | null;
```

Reply:
after adding the `null` assertion, the checks are unhappy about the unnecessary assertion. removing the `| null` still works and if the queryselector cannot get the element, it will still assign `null` to `cell`

Reply:
OK, then something like this would be nicer I think (using typescript type system instead of working around it)

```suggestion
        let cell = this._root.querySelector(`#${cellId}`);
```